### ISSUE TITLE
Northrend Encounters : Eye of Eternity Bugs/Glitches

### ISSUE THREAD

**Issue by MrAndersonX** [dalaranwow/dalaran-wow#3858]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (OLD - ... tags should no longer be used)
[//]: # (Priority - ... tags are assigned by team members)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Description**:

There are several glitches I experienced during the encounter like a day or so. While some are game breaking, actually preventing you from finishing the raid, some are minor.

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

1. First things first : Nexus Lord. As per the #1366  report, they do not work as they should.
"If the Nexus Lord's are Death Gripped by Death Knight when they are still flying and landing down, they will glitch out and it will be impossible to kill them. It should be the best to put immunity masks on them to prevent that. Minor glitch that should be fixed." 
 It really isn't a minor glitch. Being unable to kill a Nexus Lord, makes the raid impossible to progress to phase 2, which makes it impossible to finish the raid, either resulting in a wipe or a rest (see 2). Also, sometimes they don't fully bug out, meaning they can be killed by range only, though they still untargetable/attackable at melee range.

2. "Deep Breath animation: Currently the animation happens, but it is not on the ground but for some reason in the air at his mouth directly, like there is invisible barrier up there for some reason. Most likely invisible world barrier at coordinate Z is glitching this out, 99% sure. This doesn't work as intended, the breath should hit the ground. This was working on Kirin Tor just fine."
Due to this during last night's raid, we were literally unable to wipe. Even when enraged/berserk Malygos didn't do enough dmg to kill us. We stood there waiting for 15 mins trying to kill a Nexus Lord until finally we gave up. It also seemed like Malygos's skill wasn't hitting the entire platform, but only part of it.

3. "Scions of Eternity move slightly too fast than retail at the moment, I'd say 5% to 10% faster than they should. Their speed should be reduced significally.
Scions of Eternity DO NOT drop their Hover Disks at the moment which allows more players to fly around. Only Nexus Lords do that at the moment. This doesn't work as intended since they should make them available and usable for players, correct me if I'm wrong." 
This makes melee almost useless on the ground and further forces the group to speed things up to stop Malygos from gaining Enrage (even though his dmg is bugged in a negative way). 

4. "If a player on a dragon dies, he will fall down into abyss until he dies, his release button will however be grayed out, so he has to do /reload to release." 
Extremly annoying bug.

5. "Portal spawning / despawning: Currently the portal spawns which allows players to exit or enter the instance after Malygos's death. The issue is that is usable only ONCE by someone, and after that first usage, it will despawn. That doesn't work as intended, the portal should stay spawned forever after Malygos's death.." 
Another minor/major issue, depending on a player's point of view.

6. "When a player will try to return to the instance, and moves into the instance (after Malygos is dead on that ID), he should be automatically switched on the flying drake so he doens't fall into oblivion."
This is a major issue that forces players to open/submit tickets asking for the loot as they cannot obtain it themselves after they die, as for the quest item loot.

**Expected behaviour**: Tell us what should happen instead

1.  They should not bug out/partially bug out if they are Death Griped.

2.  His dmg should be fixed, so if players don't stay under the protective dome/buble, they will get severly hurt by his dmg or even killed, and certainly killed by his dmg when he is Enraged.

3. They should move at normal speed.

4. After a player dies/falls from his/hers dragon, he should be able to release, without being force to reload to release.

5. The portal should stay forever allowing all raid members to exit the area without despawning.

6. A dragon should spawn when raid members that died and respawned want to enter the raid area again to collet his/hers quest reward/loot. They should not fall to their deaths.

**Steps to reproduce the problem**:

1.  Enter the raid area.
2.  Test the above matters.
3. 

**Include proofs for this behaviour**

https://postimg.org/gallery/1p8w09au2/

 As you can see from these 2 ps, the Nexus Lord is unattackable, Malygos's enrage meter was gone, because he was Enraged, yet even those of us outside of the domes/bubbles are still well and alive.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

Nexus Lord : http://www.wowhead.com/npc=30245/nexus-lord
Malygos : http://www.wowhead.com/npc=28859/malygos
Deep Breath + other abilities/tactics/how they should work : http://wowwiki.wikia.com/wiki/Malygos_(tactics)

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

Alot of problems related to this encounter are also related in ticket number #1366.

http://www.wowhead.com/the-eye-of-eternity
Zone ID : 4500
 
**Include Screenshots from the issue if necessary**

**Comment by Crispybuilt** [comment]
Also I think you forgot to add the issue about the vortex from the previous report:

- Vortex which should pick up the entire raid only picks up one person at the moment. He is getting damaged as he should and the player can cast instant spells as intended.

How it should work: http://youtu.be/J81w3FFcojs?t=3m31s
How does it work at the moment: http://youtu.be/3-u5v2RwC5U?t=50s
Vortex should drop a player down when it's over and roughly do 10 000 damage, that currently works as intended.
After the Vortex is and animation is over, all the players will lose combat and re-enter the next second for some reason, making this slightly easier, since things such as potions can be used again. That should currently not be the case and should not happen.
Vortex should be delayed a bit (starts 3 seconds after Malygos flies to center, not instant like here on Dalaran WoW) before starting to being active: This is due the quote he says before picking up players (check quote report down below). This is not working as intended.

**Comment by MrAndersonX** [comment]
@Crispybuilt Yeah ty, was late at night and didn't notice :).

**Comment by Crispybuilt** [comment]
**Description**: During Phase 3 of Malygos encounter in The Eye Of Eternity raid instance Wyrmrest Skytalon's ability Flame Shield can never be cast.

**Current behaviour**:
During [Phase 3](http://wowwiki.wikia.com/wiki/Malygos_(tactics)#Phase_Three) of [Malygos](http://www.wowhead.com/npc=28859/malygos) encounter in [The Eye Of Eternity](http://www.wowhead.com/the-eye-of-eternity) raid instance [Wyrmrest Skytalon's](http://www.wowhead.com/npc=32535/wyrmrest-skytalon) ability [Flame Shield](http://www.wowhead.com/spell=57108/flame-shield) can never be cast.

An error message "No target" is displayed instead.

**Expected behaviour**: [Flame Shield](http://www.wowhead.com/spell=57108/flame-shield) should be cast and a buff [Flame Shield](http://www.wowhead.com/spell=75926/flame-shield) should appear on your [Wyrmrest Skytalon](http://www.wowhead.com/npc=32535/wyrmrest-skytalon) reducing damage taken by 80% and lasting 2/3/4/5/6 seconds depending on how many combo points were consumed.

**Steps to reproduce the problem**:

Reproduced in both situations:
* Target [Malygos](http://www.wowhead.com/npc=28859/malygos), cast [Flame Spike](http://www.wowhead.com/spell=56091/flame-spike) to build at least one combo point on him, make sure your [Wyrmrest Skytalon](http://www.wowhead.com/npc=32535/wyrmrest-skytalon) has at least 25 energy and then try to cast [Flame Shield](http://www.wowhead.com/spell=57108/flame-shield) while [Malygos](http://www.wowhead.com/npc=28859/malygos) is still targeted.
* Target any [Wyrmrest Skytalon](http://www.wowhead.com/npc=32535/wyrmrest-skytalon) (including your own) from your group, cast [Revivify](http://www.wowhead.com/spell=57090/revivify) to build at least one combo point on him, make sure your [Wyrmrest Skytalon](http://www.wowhead.com/npc=32535/wyrmrest-skytalon) has at least 25 energy and then try to cast [Flame Shield](http://www.wowhead.com/spell=57108/flame-shield) while selected [Wyrmrest Skytalon](http://www.wowhead.com/npc=32535/wyrmrest-skytalon) from your group is still targeted.

**Include proofs for this behaviour**

* http://wowwiki.wikia.com/wiki/Malygos_(tactics)#Phase_Three
* http://www.wowhead.com/npc=28859/malygos#article

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

* [The Eye Of Eternity](http://www.wowhead.com/the-eye-of-eternity)
* [Malygos](http://www.wowhead.com/npc=28859/malygos) - 28859
* [Wyrmrest Skytalon](http://www.wowhead.com/npc=32535/wyrmrest-skytalon) - 32535
* [Flame Spike](http://www.wowhead.com/spell=56091/flame-spike) - 56091
* [Revivify](http://www.wowhead.com/spell=57090/revivify) - 57090
* [Flame Shield](http://www.wowhead.com/spell=57108/flame-shield) - 57108
* [Flame Shield](http://www.wowhead.com/spell=75926/flame-shield) (buff) - 75926

**Include Screenshots from the issue if necessary**

Screenshot was not taken during Malygos encounter itself, but behavior is exactly the same.
![wowscrnshot_112316_012829](https://cloud.githubusercontent.com/assets/9550798/20546163/326b19c0-b11c-11e6-9efd-ee6f2b5c34d0.jpg)

**Comment by Zelwindin** [comment]
:+1:

**Comment by Rushor** [comment]
@Zelwindin adding the thumb up i the startpost would be enough :)

**Comment by datbunneh** [comment]
One additional issue:
1. Sparks are moving during Vortex. They should be paused until the raid is released from the Vortex. After they are unpaused they should be moving a bit faster.
2. Sparks should be constantly moving towards the boss and do not agrro any players. Currently you can damage a spark and it will stay by you as long as you're the 1st on its threat table. However they should remain snarable or grippable.
3. Sparks are getting consumed by the boss even when they have not reached him. Looks like there is some sort of timer or something else.
4. As far as I can tell from the Tankspot's guide, the field buffing any raid members standing in it left by a killed spark should last longer potentially letting you stack multiple fields in the same spot and getting additional stacks of the buff.
5. Sparks do not always instantly spawn after announcement. Looks like they sometimes spawn a bit too far away from the platform which takes them more time to travel to players.

**Comment by alexuktc** [comment]
Additional issue I encountered today:

Players on discs get damaged by Deep Breath. This should not happen. No Skill should hit them while on the discs.

**Comment by datbunneh** [comment]
Also Saronite Bombs instantly destroy floor on p1/p2.

**Comment by TeridanDW** [comment]
Regarding:

> Also Saronite Bombs instantly destroy floor on p1/p2.

Possible solution might be related to the ground floor in ToC where the bombs does NOT break it.
